# Goal

# Slides to show
tum_ar_window/ARSlide[] slides

# alternatively: name of file to load slides from
string task_description_file

---
# Result

# Result of inspection
tum_ar_window/InspectionResult result

---
# Feedback

# Number of Slide that has been inspected
uint32 slide

# Number of checks that need to be performed
uint32 slides

# Result of inspection
tum_ar_window/InspectionResult result